### PR TITLE
feat(profiling): add platform specific docs

### PR DESCRIPTION
**Pull request by JonasBa**
Adds platform specific docs for browser profiling

### CODE REVIEW

File: src/platform-includes/profiling/automatic-instrumentation-setup/javascript.angular.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+```javascript
+import * as Sentry from "@sentry/angular";
```
Comment (AbhiPrasad) [inline src/platform-includes/profiling/automatic-instrumentation-setup/javascript.angular.mdx:2]:
```suggestion
import * as Sentry from "@sentry/angular-ivy";
```

File: src/platform-includes/profiling/automatic-instrumentation-intro/javascript.angular.mdx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+```bash
+# Using yarn
+yarn add @sentry/angular
+
+# Using npm
+npm install --save @sentry/angular
```
Comment (AbhiPrasad) [inline src/platform-includes/profiling/automatic-instrumentation-intro/javascript.angular.mdx:6]:
```suggestion
yarn add @sentry/angular-ivy

# Using npm
npm install --save @sentry/angular-ivy
```

**Review by AbhiPrasad — APPROVED** [review]
:shipit:

File: src/platform-includes/profiling/automatic-instrumentation-intro/_default.mdx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+```bash
+# Using yarn
+yarn add @sentry/browser
+
+# Using npm
+npm install --save @sentry/browser
+```
```
Comment (shanamatthews) [inline src/platform-includes/profiling/automatic-instrumentation-intro/_default.mdx:7]: Can we use code tabs for these instead? Also, I think it's preferred to put npm first: https://github.com/getsentry/sentry-docs/pull/8362

```suggestion
```bash {tabTitle:npm}
npm install --save @sentry/browser
```

```bash {tabTitle:yarn}
yarn add @sentry/browser
```
```

Comment (shanamatthews) [reply src/platform-includes/profiling/automatic-instrumentation-intro/_default.mdx:7]: ahh whoa sorry github really doesn't like fenced code blocks in suggestions.

If you edit this comment you can grab what it should be


```bash {tabTitle:npm}
npm install --save @sentry/browser
```

```bash {tabTitle:yarn}
yarn add @sentry/browser
```

Comment (JonasBa) [reply src/platform-includes/profiling/automatic-instrumentation-intro/_default.mdx]: Oh yes, updated!

**Review by shanamatthews — APPROVED** [review]
LGTM! Thanks for making that adjustment.

**Pull request merged**